### PR TITLE
Set up karma unit tests with gulp

### PR DESCRIPTION
**Pull request by malucky**
@deini @alexmckenley

### CODE REVIEW

**Comment by deini** [comment]
Damn that was fast

File: package.json [hunk 1]
```diff
@@ -20,7 +20,14 @@
     "gulp-concat-css": "^0.1.4",
     "gulp-filter": "^1.0.0",
     "gulp-git": "^0.4.3",
+    "gulp-jasmine": "^1.0.1",
+    "gulp-karma": "0.0.4",
     "gulp-tag-version": "^1.0.2",
+    "karma": "^0.12.23",
+    "karma-chrome-launcher": "^0.1.4",
```
Comment (alexmckenley) [inline package.json]: Looks like we're not using chrome anywhere, should this be removed for now?

**Comment by alexmckenley** [comment]
:+1:  Looks good. Thanks for doing this @malucky

**Comment by deini** [comment]
LGTM :+1:

**Pull request merged**